base/Base.jl

```diff
@@ -364,7 +364,7 @@ end
 # These functions are duplicated in client.jl/include(::String) for
 # nicer stacktraces. Modifications here have to be backported there
 include(mod::Module, path::AbstractString) = include(mod, convert(String, path))
-function include(mod::Module, _path::String)
+function include(mod::Module, _path::String; mapexpr::Function=identity)
```

Comment:
I realize this is a bit of a corner case to call it like this, but perhaps the function argument should be the optional first one?

Reply:
Sure, that might be nice.